### ISSUE TITLE
Self-signed certificate support is not working

### ISSUE THREAD

**Issue by sanyarnd** [iwalton3/jellyfin-mpv-shim#134]
I'm running Jellyfin server behind a reverse proxy with a self-signed SSL certificate. Web version works fine, android app as well (since version 2). 

I see that there is an option `ignore_ssl_cert`, but it's not working for some reason?

Server URL: https://673706cc013b.sn.mynetname.net:4444/jellyfin

**Comment by sanyarnd** [comment]
Nevermind, I tried to find my answer, but was using wrong keywords. 

I see that there is an option `ignore_ssl_cert`.

**Comment by iwalton3** [comment]
To confirm, you did restart the application after changing the config file, right? Were there any [log messages](https://github.com/iwalton3/jellyfin-mpv-shim/wiki/Sending-Logs)?

**Comment by sanyarnd** [comment]
Yes, I did. Server is now offline due to power outage. I'll try to bring it back online tomorrow and will send the logs.

**Comment by sanyarnd** [comment]
<details>
<summary>log</summary>

```log
2020-11-20 02:00:02,212 [    INFO] player: Using libmpv1 playback backend.
2020-11-20 02:00:02,298 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.client: JellyfinClient initializing...
2020-11-20 02:00:02,299 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Configuration initializing...
2020-11-20 02:00:02,299 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Begin http constructor.
2020-11-20 02:00:02,300 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.ws_client: WSClient initializing...
2020-11-20 02:00:02,300 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: ConnectionManager initializing...
2020-11-20 02:00:02,301 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.credentials: Credentials initializing...
2020-11-20 02:00:02,304 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Begin app constructor.
2020-11-20 02:00:02,304 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Begin connect
2020-11-20 02:00:02,305 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Begin getAvailableServers
2020-11-20 02:00:02,306 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: MultiGroup      : ('<broadcast>', 7359)
2020-11-20 02:00:02,306 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: Sending UDP Data: b'who is JellyfinServer?'
2020-11-20 02:00:03,310 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Found Servers: []
2020-11-20 02:00:03,312 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.credentials: credentialsupdated
2020-11-20 02:00:03,313 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: connect has 1 servers
2020-11-20 02:00:03,314 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: begin connectToServer
2020-11-20 02:00:03,314 [    INFO] JELLYFIN.jellyfin_apiclient_python.api: Sending get request to system/info/public
2020-11-20 02:00:03,315 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: 5
2020-11-20 02:00:03,316 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: {'Accept': 'application/json', 'Content-type': 'application/x-www-form-urlencoded; charset=UTF-8', 'X-Application': 'Jellyfin MPV Shim/1.8.0', 'Accept-Charset': 'UTF-8,*', 'Accept-encoding': 'gzip', 'User-Agent': 'Jellyfin-MPV-Shim/1.8.0', 'x-emby-authorization': 'MediaBrowser Client=Jellyfin MPV Shim, Device=fujitsu-e558, DeviceId=e5a8053a-9471-4f30-a5ab-9cc328cfe936, Version=1.8.0'}
2020-11-20 02:00:03,327 [   DEBUG] urllib3.connectionpool: Starting new HTTPS connection (1): 673706cc013b.sn.mynetname.net:4444
2020-11-20 02:00:03,603 [   DEBUG] urllib3.connectionpool: https://673706cc013b.sn.mynetname.net:4444 "GET /jellyfin//system/info/public HTTP/1.1" 200 172
2020-11-20 02:00:03,625 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: calling onSuccessfulConnection with server Jellyfin
2020-11-20 02:00:03,627 [    INFO] JELLYFIN.jellyfin_apiclient_python.api: Sending get request to system/info
2020-11-20 02:00:03,627 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: 5
2020-11-20 02:00:03,629 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: {'Accept': 'application/json', 'Content-type': 'application/x-www-form-urlencoded; charset=UTF-8', 'X-Application': 'Jellyfin MPV Shim/1.8.0', 'Accept-Charset': 'UTF-8,*', 'Accept-encoding': 'gzip', 'User-Agent': 'Jellyfin-MPV-Shim/1.8.0', 'x-emby-authorization': 'MediaBrowser Client=Jellyfin MPV Shim, Device=fujitsu-e558, DeviceId=e5a8053a-9471-4f30-a5ab-9cc328cfe936, Version=1.8.0', 'X-MediaBrowser-Token': 'REDACTED'}
2020-11-20 02:00:03,642 [   DEBUG] urllib3.connectionpool: Starting new HTTPS connection (1): 673706cc013b.sn.mynetname.net:4444
2020-11-20 02:00:03,827 [   DEBUG] urllib3.connectionpool: https://673706cc013b.sn.mynetname.net:4444 "GET /jellyfin//system/info HTTP/1.1" 200 437
2020-11-20 02:00:03,835 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.credentials: credentialsupdated
2020-11-20 02:00:03,835 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: resolving connect with result: {'Servers': "[{'address': 'https://673706cc013b.sn.mynetname.net:4444/jellyfin/', 'Name': 'Jellyfin', 'Id': '0d8c12e693ee465fa85b077fd3574340', 'DateLastAccessed': '2020-11-20T02:00:03Z', 'UserId': '99f6d97dcaf44e2ca770e894e421e329', 'AccessToken': 'REDACTED', 'Users': [{'Id': '99f6d97dcaf44e2ca770e894e421e329', 'IsSignedInOffline': True}], 'uuid': '43dc797d-5237-4033-ad3b-fe9e2dfd7e15', 'username': 'alex', 'connected': False}]", 'State': 3}
2020-11-20 02:00:03,836 [    INFO] JELLYFIN.jellyfin_apiclient_python.client: User is authenticated.
2020-11-20 02:00:03,836 [   DEBUG] urllib3.util.retry: Converted retries value: 3 -> 'Retry(total=3, connect=None, read=None, redirect=None, status=None)'
2020-11-20 02:00:03,836 [   DEBUG] urllib3.util.retry: Converted retries value: 3 -> 'Retry(total=3, connect=None, read=None, redirect=None, status=None)'
2020-11-20 02:00:03,837 [    INFO] JELLYFIN.jellyfin_apiclient_python.ws_client: Websocket url: wss://673706cc013b.sn.mynetname.net:4444/jellyfin//socket?api_key=REDACTEDREDACTED&device_id=e5a8053a-9471-4f30-a5ab-9cc328cfe936
2020-11-20 02:00:03,842 [   DEBUG] urllib3.connectionpool: Starting new HTTPS connection (1): 673706cc013b.sn.mynetname.net:4444
2020-11-20 02:00:03,990 [   ERROR] JELLYFIN.jellyfin_apiclient_python.ws_client: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1076)
2020-11-20 02:00:03,991 [   DEBUG] event_handler: Unhandled Event WebSocketError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1076)
2020-11-20 02:00:03,993 [    INFO] JELLYFIN.jellyfin_apiclient_python.ws_client: ---<[ websocket ]
2020-11-20 02:00:03,994 [    INFO] clients: No connection to server. Next try in 1 second(s)
2020-11-20 02:00:03,994 [    INFO] Jellyfin.jellyfin_apiclient_python.http: --<[ session/1898944976712 ]
2020-11-20 02:00:04,028 [   DEBUG] urllib3.connectionpool: https://673706cc013b.sn.mynetname.net:4444 "POST /jellyfin//Sessions/Capabilities/Full HTTP/1.1" 204 0
2020-11-20 02:00:04,995 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.client: JellyfinClient initializing...
2020-11-20 02:00:04,995 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Configuration initializing...
2020-11-20 02:00:04,996 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Begin http constructor.
2020-11-20 02:00:04,996 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.ws_client: WSClient initializing...
2020-11-20 02:00:04,996 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: ConnectionManager initializing...
2020-11-20 02:00:04,997 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.credentials: Credentials initializing...
2020-11-20 02:00:04,997 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Begin app constructor.
2020-11-20 02:00:04,998 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Begin connect
2020-11-20 02:00:04,998 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Begin getAvailableServers
2020-11-20 02:00:04,999 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: MultiGroup      : ('<broadcast>', 7359)
2020-11-20 02:00:04,999 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: Sending UDP Data: b'who is JellyfinServer?'
2020-11-20 02:00:06,000 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Found Servers: []
2020-11-20 02:00:06,001 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.credentials: credentialsupdated
2020-11-20 02:00:06,002 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: connect has 1 servers
2020-11-20 02:00:06,002 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: begin connectToServer
2020-11-20 02:00:06,003 [    INFO] JELLYFIN.jellyfin_apiclient_python.api: Sending get request to system/info/public
2020-11-20 02:00:06,004 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: 5
2020-11-20 02:00:06,004 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: {'Accept': 'application/json', 'Content-type': 'application/x-www-form-urlencoded; charset=UTF-8', 'X-Application': 'Jellyfin MPV Shim/1.8.0', 'Accept-Charset': 'UTF-8,*', 'Accept-encoding': 'gzip', 'User-Agent': 'Jellyfin-MPV-Shim/1.8.0', 'x-emby-authorization': 'MediaBrowser Client=Jellyfin MPV Shim, Device=fujitsu-e558, DeviceId=e5a8053a-9471-4f30-a5ab-9cc328cfe936, Version=1.8.0'}
2020-11-20 02:00:06,012 [   DEBUG] urllib3.connectionpool: Starting new HTTPS connection (1): 673706cc013b.sn.mynetname.net:4444
2020-11-20 02:00:06,274 [   DEBUG] urllib3.connectionpool: https://673706cc013b.sn.mynetname.net:4444 "GET /jellyfin//system/info/public HTTP/1.1" 200 172
2020-11-20 02:00:06,286 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: calling onSuccessfulConnection with server Jellyfin
2020-11-20 02:00:06,287 [    INFO] JELLYFIN.jellyfin_apiclient_python.api: Sending get request to system/info
2020-11-20 02:00:06,288 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: 5
2020-11-20 02:00:06,289 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: {'Accept': 'application/json', 'Content-type': 'application/x-www-form-urlencoded; charset=UTF-8', 'X-Application': 'Jellyfin MPV Shim/1.8.0', 'Accept-Charset': 'UTF-8,*', 'Accept-encoding': 'gzip', 'User-Agent': 'Jellyfin-MPV-Shim/1.8.0', 'x-emby-authorization': 'MediaBrowser Client=Jellyfin MPV Shim, Device=fujitsu-e558, DeviceId=e5a8053a-9471-4f30-a5ab-9cc328cfe936, Version=1.8.0', 'X-MediaBrowser-Token': 'REDACTED'}
2020-11-20 02:00:06,296 [   DEBUG] urllib3.connectionpool: Starting new HTTPS connection (1): 673706cc013b.sn.mynetname.net:4444
2020-11-20 02:00:06,522 [   DEBUG] urllib3.connectionpool: https://673706cc013b.sn.mynetname.net:4444 "GET /jellyfin//system/info HTTP/1.1" 200 437
2020-11-20 02:00:06,527 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.credentials: credentialsupdated
2020-11-20 02:00:06,528 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: resolving connect with result: {'Servers': "[{'address': 'https://673706cc013b.sn.mynetname.net:4444/jellyfin/', 'Name': 'Jellyfin', 'Id': '0d8c12e693ee465fa85b077fd3574340', 'DateLastAccessed': '2020-11-20T02:00:06Z', 'UserId': '99f6d97dcaf44e2ca770e894e421e329', 'AccessToken': 'REDACTED', 'Users': [{'Id': '99f6d97dcaf44e2ca770e894e421e329', 'IsSignedInOffline': True}], 'uuid': '43dc797d-5237-4033-ad3b-fe9e2dfd7e15', 'username': 'alex', 'connected': False}]", 'State': 3}
2020-11-20 02:00:06,528 [    INFO] JELLYFIN.jellyfin_apiclient_python.client: User is authenticated.
2020-11-20 02:00:06,529 [   DEBUG] urllib3.util.retry: Converted retries value: 3 -> 'Retry(total=3, connect=None, read=None, redirect=None, status=None)'
2020-11-20 02:00:06,529 [   DEBUG] urllib3.util.retry: Converted retries value: 3 -> 'Retry(total=3, connect=None, read=None, redirect=None, status=None)'
2020-11-20 02:00:06,530 [    INFO] JELLYFIN.jellyfin_apiclient_python.ws_client: Websocket url: wss://673706cc013b.sn.mynetname.net:4444/jellyfin//socket?api_key=REDACTEDREDACTED&device_id=e5a8053a-9471-4f30-a5ab-9cc328cfe936
2020-11-20 02:00:06,534 [   DEBUG] urllib3.connectionpool: Starting new HTTPS connection (1): 673706cc013b.sn.mynetname.net:4444
2020-11-20 02:00:06,902 [   ERROR] JELLYFIN.jellyfin_apiclient_python.ws_client: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1076)
2020-11-20 02:00:06,902 [   DEBUG] event_handler: Unhandled Event WebSocketError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1076)
2020-11-20 02:00:06,908 [    INFO] JELLYFIN.jellyfin_apiclient_python.ws_client: ---<[ websocket ]
2020-11-20 02:00:06,908 [    INFO] clients: No connection to server. Next try in 1 second(s)
2020-11-20 02:00:06,909 [    INFO] Jellyfin.jellyfin_apiclient_python.http: --<[ session/1898910349640 ]
2020-11-20 02:00:06,937 [   DEBUG] urllib3.connectionpool: https://673706cc013b.sn.mynetname.net:4444 "POST /jellyfin//Sessions/Capabilities/Full HTTP/1.1" 204 0
2020-11-20 02:00:07,912 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.client: JellyfinClient initializing...
2020-11-20 02:00:07,913 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Configuration initializing...
2020-11-20 02:00:07,914 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Begin http constructor.
2020-11-20 02:00:07,915 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.ws_client: WSClient initializing...
2020-11-20 02:00:07,916 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: ConnectionManager initializing...
2020-11-20 02:00:07,916 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.credentials: Credentials initializing...
2020-11-20 02:00:07,917 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Begin app constructor.
2020-11-20 02:00:07,918 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Begin connect
2020-11-20 02:00:07,919 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Begin getAvailableServers
2020-11-20 02:00:07,920 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: MultiGroup      : ('<broadcast>', 7359)
2020-11-20 02:00:07,921 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: Sending UDP Data: b'who is JellyfinServer?'
2020-11-20 02:00:08,923 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Found Servers: []
2020-11-20 02:00:08,925 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.credentials: credentialsupdated
2020-11-20 02:00:08,926 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: connect has 1 servers
2020-11-20 02:00:08,927 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: begin connectToServer
2020-11-20 02:00:08,928 [    INFO] JELLYFIN.jellyfin_apiclient_python.api: Sending get request to system/info/public
2020-11-20 02:00:08,929 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: 5
2020-11-20 02:00:08,929 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: {'Accept': 'application/json', 'Content-type': 'application/x-www-form-urlencoded; charset=UTF-8', 'X-Application': 'Jellyfin MPV Shim/1.8.0', 'Accept-Charset': 'UTF-8,*', 'Accept-encoding': 'gzip', 'User-Agent': 'Jellyfin-MPV-Shim/1.8.0', 'x-emby-authorization': 'MediaBrowser Client=Jellyfin MPV Shim, Device=fujitsu-e558, DeviceId=e5a8053a-9471-4f30-a5ab-9cc328cfe936, Version=1.8.0'}
2020-11-20 02:00:08,944 [   DEBUG] urllib3.connectionpool: Starting new HTTPS connection (1): 673706cc013b.sn.mynetname.net:4444
2020-11-20 02:00:09,170 [   DEBUG] urllib3.connectionpool: https://673706cc013b.sn.mynetname.net:4444 "GET /jellyfin//system/info/public HTTP/1.1" 200 172
2020-11-20 02:00:09,180 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: calling onSuccessfulConnection with server Jellyfin
2020-11-20 02:00:09,180 [    INFO] JELLYFIN.jellyfin_apiclient_python.api: Sending get request to system/info
2020-11-20 02:00:09,181 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: 5
2020-11-20 02:00:09,181 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.api: {'Accept': 'application/json', 'Content-type': 'application/x-www-form-urlencoded; charset=UTF-8', 'X-Application': 'Jellyfin MPV Shim/1.8.0', 'Accept-Charset': 'UTF-8,*', 'Accept-encoding': 'gzip', 'User-Agent': 'Jellyfin-MPV-Shim/1.8.0', 'x-emby-authorization': 'MediaBrowser Client=Jellyfin MPV Shim, Device=fujitsu-e558, DeviceId=e5a8053a-9471-4f30-a5ab-9cc328cfe936, Version=1.8.0', 'X-MediaBrowser-Token': 'REDACTED'}
2020-11-20 02:00:09,185 [   DEBUG] urllib3.connectionpool: Starting new HTTPS connection (1): 673706cc013b.sn.mynetname.net:4444
2020-11-20 02:00:09,447 [   DEBUG] urllib3.connectionpool: https://673706cc013b.sn.mynetname.net:4444 "GET /jellyfin//system/info HTTP/1.1" 200 437
2020-11-20 02:00:09,467 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.credentials: credentialsupdated
2020-11-20 02:00:09,468 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: resolving connect with result: {'Servers': "[{'address': 'https://673706cc013b.sn.mynetname.net:4444/jellyfin/', 'Name': 'Jellyfin', 'Id': '0d8c12e693ee465fa85b077fd3574340', 'DateLastAccessed': '2020-11-20T02:00:09Z', 'UserId': '99f6d97dcaf44e2ca770e894e421e329', 'AccessToken': 'REDACTED', 'Users': [{'Id': '99f6d97dcaf44e2ca770e894e421e329', 'IsSignedInOffline': True}], 'uuid': '43dc797d-5237-4033-ad3b-fe9e2dfd7e15', 'username': 'alex', 'connected': False}]", 'State': 3}
2020-11-20 02:00:09,469 [    INFO] JELLYFIN.jellyfin_apiclient_python.client: User is authenticated.
2020-11-20 02:00:09,470 [   DEBUG] urllib3.util.retry: Converted retries value: 3 -> 'Retry(total=3, connect=None, read=None, redirect=None, status=None)'
2020-11-20 02:00:09,471 [   DEBUG] urllib3.util.retry: Converted retries value: 3 -> 'Retry(total=3, connect=None, read=None, redirect=None, status=None)'
2020-11-20 02:00:09,473 [    INFO] JELLYFIN.jellyfin_apiclient_python.ws_client: Websocket url: wss://673706cc013b.sn.mynetname.net:4444/jellyfin//socket?api_key=REDACTEDREDACTED&device_id=e5a8053a-9471-4f30-a5ab-9cc328cfe936
2020-11-20 02:00:09,480 [   DEBUG] urllib3.connectionpool: Starting new HTTPS connection (1): 673706cc013b.sn.mynetname.net:4444
2020-11-20 02:00:09,677 [   ERROR] JELLYFIN.jellyfin_apiclient_python.ws_client: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1076)
2020-11-20 02:00:09,679 [   DEBUG] event_handler: Unhandled Event WebSocketError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1076)
2020-11-20 02:00:09,689 [    INFO] JELLYFIN.jellyfin_apiclient_python.ws_client: ---<[ websocket ]
2020-11-20 02:00:09,693 [    INFO] clients: No connection to server. Next try in 1 second(s)
2020-11-20 02:00:09,695 [    INFO] Jellyfin.jellyfin_apiclient_python.http: --<[ session/1898944861448 ]
2020-11-20 02:00:09,733 [   DEBUG] urllib3.connectionpool: https://673706cc013b.sn.mynetname.net:4444 "POST /jellyfin//Sessions/Capabilities/Full HTTP/1.1" 204 0
2020-11-20 02:00:10,698 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.client: JellyfinClient initializing...
2020-11-20 02:00:10,699 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Configuration initializing...
2020-11-20 02:00:10,700 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Begin http constructor.
2020-11-20 02:00:10,701 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.ws_client: WSClient initializing...
2020-11-20 02:00:10,701 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: ConnectionManager initializing...
2020-11-20 02:00:10,703 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.credentials: Credentials initializing...
2020-11-20 02:00:10,703 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.configuration: Begin app constructor.
2020-11-20 02:00:10,703 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Begin connect
2020-11-20 02:00:10,705 [    INFO] JELLYFIN.jellyfin_apiclient_python.connection_manager: Begin getAvailableServers
2020-11-20 02:00:10,705 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: MultiGroup      : ('<broadcast>', 7359)
2020-11-20 02:00:10,706 [   DEBUG] JELLYFIN.jellyfin_apiclient_python.connection_manager: Sending UDP Data: b'who is JellyfinServer?'
```

</details>

tl;dr:
```
2020-11-20 02:00:45,058 [   ERROR] JELLYFIN.jellyfin_apiclient_python.ws_client: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1076)
2020-11-20 02:00:45,059 [   DEBUG] event_handler: Unhandled Event WebSocketError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1076)
```

I have `"ignore_ssl_cert": true,` in my `conf.json`.

**Comment by iwalton3** [comment]
Ok so it looks like SSL certificate verification for the websocket library isn't being disabled. I'll have to take a look at this.

Are you on Windows or Linux?

**Comment by sanyarnd** [comment]
Windows 10

**Comment by iwalton3** [comment]
I have tested the change and verified I can now connect to bare IP addresses with HTTPS.

Please try the test build here: [Download CI Build - Installer](https://artprodcus3.artifacts.visualstudio.com/Af8b3f627-969b-4284-9f33-c9b0fdbdc993/aab8f3fe-76b5-40d1-97ad-2e95e0148cf2/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2l3YWx0b24zL3Byb2plY3RJZC9hYWI4ZjNmZS03NmI1LTQwZDEtOTdhZC0yZTk1ZTAxNDhjZjIvYnVpbGRJZC82Ny9hcnRpZmFjdE5hbWUvRGVza3RvcEluc3RhbGxlcg2/content?format=file&subPath=%2Fjellyfin-mpv-desktop_1.8.0_installer.exe)

Re-open the issue if the fixed build does not work for you.
